chain/cmd/provider/main.go

```diff
@@ -22,34 +22,78 @@ import (
 	"github.com/bandprotocol/d3n/chain/x/zoracle"
 )
 
+type Command struct {
+	Cmd       string   `json:"cmd"`
+	Arguments []string `json:"args"`
+}
+
 const limitTimeOut = 10 * time.Second
 
 // TODO: Replace by `BandStatefulClient` after implementation finished.
 var bandProvider d3nlib.BandProvider
 var allowedCommands = map[string]bool{"curl": true, "date": true}
 
-func main() {
-	// Get environment variable
-	privS, ok := os.LookupEnv("PRIVATE_KEY")
-	if !ok {
-		log.Fatal("Missing private key")
+func getEnv(key, def string) string {
```

Comment:
let's spell it out `default`. I was confused at first with this `def` thinking it's like Python's `def` lol